openshift/secrets-template.yaml

```diff
@@ -41,6 +41,7 @@ objects:
   data:
     aws_access_key_id: ${AWS_ACCESS_KEY_ID}
     aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_region: ${AWS_DEFAULT_REGION}
```

Comment:
This region is required for EMR and SQS also. Can you please make that change?

Reply:
Added aws region to all these and also in 'aws' secret.